CONTRIBUTING.md

```diff
@@ -76,3 +76,128 @@ issues.
+          ├─ parse_iso_intervals.c  # Generated by re2c 0.15.3
+          └─ ...
+       └─ ...
+    └─ fileinfo/
+       ├─ libmagic/                 # Modified libmagic https://github.com/file/file
+       ├─ data_file.c               # Generated by `ext/fileinfo/create_data_file.php`
+       ├─ libmagic.patch            # Modifications patch from upstream libmagic
+       ├─ magicdata.patch           # Modifications patch from upstream libmagic
+       └─ ...
+    └─ gd/
+       ├─ libgd/                    # Bundled and modified GD library https://github.com/libgd/libgd
+       └─ ...
+    └─ json/
+       ├─ json_parser.tab.c         # Generated by bison
+       ├─ json_parser.tab.h         # Generated by bison
+       ├─ json_scanner.c            # Generated by re2c 1.0.3
+       ├─ php_json_scanner_defs.h   # Generated by re2c 1.0.3
+       └─ ...
+    └─ mbstring/
+       ├─ libmbfl/                  # Forked and maintained in php-src
+       ├─ oniguruma/                # Bundled https://github.com/kkos/oniguruma
+       ├─ oniguruma.patch           # Modifications patch from upstream oniguruma
+       ├─ unicode_data.h            # Generated by `ext/mbstring/ucgendat/ucgendat.php`
+       └─ ...
+    └─ pcre/
+       ├─ pcre2lib/                 # https://www.pcre.org/
+       └─ ...
+    └─ pdo/
+       ├─ pdo_sql_parser.c          # Generated by re2c 1.0.3
+       └─ ...
+    └─ phar/
+       ├─ phar_path_check.c         # Generated by re2c 1.0.3
+       └─ ...
+    └─ skeleton/                    # Skeleton for developing new extensions with `ext/ext_skel.php`
+       └─ ...
+    └─ sqlite3/
+       ├─ libsqlite/                # https://www.sqlite.org mirror: https://github.com/mackyle/sqlite
+       └─ ...
+    └─ standard/
+       └─ html_tables/
+          ├─ mappings/              # https://www.unicode.org/Public/MAPPINGS/
+          └─ ...
+       ├─ credits_ext.h             # Generated by `scripts/dev/credits`
+       ├─ credits_sapi.h            # Generated by `scripts/dev/credits`
+       ├─ html_tables.h             # Generated by `ext/standard/html_tables/html_table_gen.php`
+       ├─ url_scanner_ex.c          # Generated by re2c 1.0.3
+       ├─ var_unserializer.c        # Generated by re2c 1.0.3
+       └─ ...
+    └─ tokenizer/
+       ├─ tokenizer_data.c          # Generated by `ext/tokenizer/tokenizer_data_gen.sh`
+       └─ ...
+    └─ xmlrpc/
+       ├─ libxmlrpc/                # Forked and maintained in php-src
+       └─ ...
+    └─ zend_test                    # For testing internal APIs. Not needed for regular builds.
+       └─ ...
+    └─ zip/                         # Bundled https://github.com/pierrejoye/php_zip
+       └─ ...
+    └─ ...
+ └─ main/                           # Binding that ties extensions, SAPIs, and engine together
+    ├─ streams/                     # Streams layer subsystem
+    ├─ php_version.h                # Generated by `configure`
```

Comment:
Some users, like @rlerdorf [has karma](https://people.php.net/rasmus) to edit this file. Is that karma outdated?

Reply:
This is part of the generated files that we keep in git. The separate karma restriction is actually there to avoid some accidental merges in the wrong direction. If someone merges master into PHP-7.3, they are going to modify the php_version.h file, which will be rejected by the karma checks. It's a bit of a hack ;)

Reply:
@carusogabriel, as for karma, any PHP Group and active/past RMs do have access to this but yeah its a bit of a hack

Reply:
Thanks for this info... I've mentioned the RMs in the comment to better understand it.